osbuild/util/runners.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/libexec/platform-python
```

Comment:
(nitpick) it seems most of the other files in osbuild/util do not have a `'!/...` and given that this is not run directly(?) it could be ommited here as well?

Reply:
Also, we shouldn't be using `/usr/libexec/platform-python` at all. If you do use a `#!` line it should be `#!/usr/bin/python3` -- the platform-python conversion is done when a rpm is built on a system using a fixed python version for the os tools (eg. RHEL, CentOS, etc.).

Reply:
Yea, though I'm a bit iffy on if it should link directly to `/usr/bin` or go through `/usr/bin/env` for those of us who run our code in virtual environments; I'm reasonably sure that one gets fixed up by RPM as well.

Reply:
As the file is not going to be run from command-line I will remove `/usr/libexec/platform-python` from it. That line was unintended .